app/server/src/main/scala/org/bitcoins/server/ServerJsonModels.scala

```diff
@@ -939,14 +982,27 @@ object DLCDataFromFile extends ServerJsonModels {
       case pathJs :: Nil =>
         Try {
           val path = new File(pathJs.str).toPath
-          DLCDataFromFile(path, None)
+          DLCDataFromFile(path, None, None, None)
         }
       case pathJs :: destJs :: Nil =>
         Try {
           val path = new File(pathJs.str).toPath
           val destJsOpt = nullToOpt(destJs)
           val destOpt = destJsOpt.map(js => new File(js.str).toPath)
-          DLCDataFromFile(path, destOpt)
+          DLCDataFromFile(path, destOpt, None, None)
+        }
+      case pathJs :: destJs :: payoutAddressJs :: changeAddressJs :: Nil =>
```

Comment:
Should also add a case for `case pathJs :: destJs :: payoutAddressJs :: Nil`

Reply:
If i understand correctly, currently we _have_ to specify the change address. I agree with Ben that that should be optional